package.json

```diff
@@ -84,11 +84,13 @@
     "@rollup/plugin-commonjs": "^18.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.0.0",
-    "@storybook/addon-essentials": "^8.0.2",
-    "@storybook/addon-links": "^8.0.2",
-    "@storybook/blocks": "^8.0.2",
-    "@storybook/web-components": "^8.0.2",
-    "@storybook/web-components-vite": "^8.0.2",
+    "@storybook/addon-essentials": "^8.2.7",
+    "@storybook/addon-links": "^8.2.7",
+    "@storybook/blocks": "^8.2.7",
+    "@storybook/manager-api": "^8.2.7",
+    "@storybook/theming": "^8.2.7",
```

Comment:
**❔ question:** Can you explain the context of why these two dependencies were not present before and added in this PR?

Reply:
These dependencies were part of `@storybook/addon-essentials` but it seems they have been moved. (cf https://github.com/storybookjs/storybook/commit/df0f14abbc59a7645a4dfc5ad3c5297b097469ae)

<details>
  <summary>npm ls</summary>

Before Storybook upgrade
```
npm ls @storybook/manager-api
@clevercloud/components@15.1.1 /home/flo-pro/Projects/clever-components
├─┬ @storybook/addon-essentials@8.0.2
│ └── @storybook/manager-api@8.0.2
├─┬ @storybook/blocks@8.0.2
│ └── @storybook/manager-api@8.0.2 deduped
├─┬ @storybook/web-components-vite@8.0.2
│ └─┬ @storybook/core-server@8.0.2
│   └── @storybook/manager-api@8.0.2 deduped
└─┬ @storybook/web-components@8.0.2
  └── @storybook/manager-api@8.0.2 deduped
```

after Storybook upgrade: 
```
❯ npm ls @storybook/manager-api
@clevercloud/components@15.1.1 /home/flo-pro/Projects/clever-components
├── @storybook/manager-api@8.2.7
└─┬ @storybook/web-components@8.2.7
  └── @storybook/manager-api@8.2.7 deduped
```

</details>


I'm don't know why and when to be honest I followed what the upgrade CLI command told me to do since it seemed to make sense.